src/renderer/components/dialogs/MailtoDialog/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import AutoSizer from 'react-virtualized-auto-sizer'
+import { C } from '@deltachat/jsonrpc-client'
+
+import ChatListItem from '../../chat/ChatListItem'
+import { PseudoListItemNoSearchResults } from '../../helpers/PseudoListItem'
+import { ChatListPart, useLogicVirtualChatList } from '../../chat/ChatList'
+import { useChatList } from '../../chat/ChatListHelpers'
+import { useThemeCssVar } from '../../../ThemeManager'
+import { selectedAccountId } from '../../../ScreenController'
+import Dialog, { DialogBody, DialogContent, DialogHeader } from '../../Dialog'
+import useCreateDraftMessage from '../../../hooks/chat/useCreateDraftMesssage'
+import useTranslationFunction from '../../../hooks/useTranslationFunction'
+
+import styles from './styles.module.scss'
+
+import type { DialogProps } from '../../../contexts/DialogContext'
+
+type Props = {
```

Comment:
TS docs recommend preferring `interface` over `type` for defining object shapes. I am not requesting any change here. But let's get back to this later. This is a reminder.

Reply:
It was there before this MR